api.yaml

```diff
@@ -1411,6 +1425,120 @@ paths:
           '429':
             $ref: '#/components/responses/TooManyRequests'
     
+    /images/screenshots/{projectId}/{testId}/{stepId}/baseline:
```

Comment:
If we're providing an endpoint for the baseline and an endpoint for the result are we deprecating the `screenshot` property from the steps on the execution?

Reply:
we plan to do that soon🤞